metadata-db-app/src/cmr/metadata_db/services/concept_service.clj

```diff
@@ -31,6 +31,8 @@
    [cmr.metadata-db.data.oracle.concepts.service]
    [cmr.metadata-db.data.oracle.concepts.tag-association]
    [cmr.metadata-db.data.oracle.concepts.tag]
+   [cmr.metadata-db.data.oracle.concepts.variable-association]
```

Comment:
These requires are needed for pulling in the defmethod definitions for the various concepts.